doc/man3/SSL_CTX_set_session_ticket_cb.pod

```diff
@@ -100,19 +100,54 @@ Same as B<SSL_TICKET_SUCCESS>, but a new ticket should be sent to the client.
 
 =back
 
+The return value can be any of these values:
+
+=over 4
+
+=item SSL_TICKET_RETURN_ERROR
+
+An error occurred.
```

Comment:
This should be more clear, perhaps even renamed to `SSL_TICKET_RETURN_ABORT`.  The idea is that the callback is choosing to abort the handshake, whether due an error in the callback or policy.

Reply:
I renamed it as suggested. Also added a few more words to describe its intended use and also a caution about the multi-tickets thing in TLSv1.3